swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md

```diff
@@ -46,15 +52,15 @@ In the terminal, execute the command below to create the Ballerina package for t
 $ bal new covid19 -t service
 ```
 
-You view the output below.
+You will see the output below.
 
-```
+```shell
```

Comment:
Since these are outputs shown in the terminal that does not need to be copied, we do not use the language type.
```suggestion
```
```